x-pack/plugins/security_solution/public/management/components/console/components/command_list.tsx

```diff
@@ -66,13 +77,27 @@ export const CommandList = memo<CommandListProps>(({ commands, display = 'defaul
 
   const footerMessage = useMemo(() => {
     return (
-      <FormattedMessage
-        id="xpack.securitySolution.console.commandList.footerText"
-        defaultMessage="For more details on the commands above use the {helpOption} argument. Example: {cmdExample}"
-        values={{
-          helpOption: <EuiCode>{'--help'}</EuiCode>,
-          cmdExample: <EuiCode>{'some-command --help'}</EuiCode>,
-        }}
+      <EuiDescriptionList
+        compressed
+        listItems={[
+          {
+            title: (
+              <StyledEuiBadge>
+                <ConsoleCodeBlock inline bold>
+                  {COMMON_ARGS.find((current) => current.name === '--help')?.name}
+                </ConsoleCodeBlock>
+              </StyledEuiBadge>
+            ),
+            description: (
+              <EuiText color="subdued" size="xs">
+                <FormattedMessage
+                  id="xpack.securitySolution.console.commandList.footerText"
+                  defaultMessage="For more details on the commands above use the --help argument. Example: some-command --help"
```

Comment:
Change this to "For more help with the individual commands use the --help argument. Ex: processes --help" to match the mocks.

https://www.figma.com/file/4U0smPyCa86uWsPn8EMFNu/Response-actions?node-id=2056%3A371963